app/src/main/res/layout/item_message_img_sent.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="8dp">
+
+    <!--    <TextView-->
```

Comment:
(This is not a change request) You can inject everything you want to comment out into a single `<!--content-->`

Or is it generated by the IDE?

Reply:
IDE did this, I deleted